Python 3.9.5 (tags/v3.9.5:0a7dcbd, May  3 2021, 17:27:52) [MSC v.1928 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license()" for more information.
>>> 
============= RESTART: C:/Users/91961/Desktop/keerthana/Day_3/4.py =============
371
Traceback (most recent call last):
  File "C:/Users/91961/Desktop/keerthana/Day_3/4.py", line 7, in <module>
    if(sum==num):
NameError: name 'num' is not defined
>>> 
============= RESTART: C:/Users/91961/Desktop/keerthana/Day_3/4.py =============
371
Traceback (most recent call last):
  File "C:/Users/91961/Desktop/keerthana/Day_3/4.py", line 3, in <module>
    while(n!=0):
NameError: name 'n' is not defined
>>> 
============= RESTART: C:/Users/91961/Desktop/keerthana/Day_3/4.py =============
371
Traceback (most recent call last):
  File "C:/Users/91961/Desktop/keerthana/Day_3/4.py", line 3, in <module>
    while(n!=0):
NameError: name 'n' is not defined
>>> 
============= RESTART: C:/Users/91961/Desktop/keerthana/Day_3/4.py =============
371
not armstrong number
not armstrong number
not armstrong number
>>> 
============= RESTART: C:/Users/91961/Desktop/keerthana/Day_3/4.py =============
371
not armstrong number
not armstrong number
not armstrong number
>>> 
============= RESTART: C:/Users/91961/Desktop/keerthana/Day_3/4.py =============
3
not armstrong number
>>> 
============= RESTART: C:/Users/91961/Desktop/keerthana/Day_3/4.py =============

============= RESTART: C:/Users/91961/Desktop/keerthana/Day_3/4.py =============
[0]
[0, 1]
[0, 1, 8]
[0, 1, 8, 27]
[0, 1, 8, 27, 64]
[0, 1, 8, 27, 64, 125]
[0, 1, 8, 27, 64, 125, 216]
[0, 1, 8, 27, 64, 125, 216, 343]
[0, 1, 8, 27, 64, 125, 216, 343, 512]
[0, 1, 8, 27, 64, 125, 216, 343, 512, 729]
[0, 1, 8, 27, 64, 125, 216, 343, 512, 729, 1000]
>>> 
============= RESTART: C:/Users/91961/Desktop/keerthana/Day_3/4.py =============
[0]
[0, 1]
[0, 1, 8]
[0, 1, 8, 27]
[0, 1, 8, 27, 64]
[0, 1, 8, 27, 64, 125]
[0, 1, 8, 27, 64, 125, 216]
[0, 1, 8, 27, 64, 125, 216, 343]
[0, 1, 8, 27, 64, 125, 216, 343, 512]
[0, 1, 8, 27, 64, 125, 216, 343, 512, 729]
[0, 1, 8, 27, 64, 125, 216, 343, 512, 729, 1000]
>>> 
============= RESTART: C:/Users/91961/Desktop/keerthana/Day_3/4.py =============
[0, 1, 8, 27, 64, 125, 216, 343, 512, 729, 1000]
>>> 
============= RESTART: C:/Users/91961/Desktop/keerthana/Day_3/4.py =============
Traceback (most recent call last):
  File "C:/Users/91961/Desktop/keerthana/Day_3/4.py", line 14, in <module>
    for i in range(n):
NameError: name 'n' is not defined
>>> 
============= RESTART: C:/Users/91961/Desktop/keerthana/Day_3/4.py =============
Traceback (most recent call last):
  File "C:/Users/91961/Desktop/keerthana/Day_3/4.py", line 14, in <module>
    for i in range(n):
NameError: name 'n' is not defined
>>> 
============= RESTART: C:/Users/91961/Desktop/keerthana/Day_3/4.py =============
10
enter the n:
enter the n:
enter the n:
enter the n:
enter the n:
enter the n:
enter the n:
enter the n:
enter the n:
enter the n:
[0, 1, 8, 27, 64, 125, 216, 343, 512, 729]
>>> 
============= RESTART: C:/Users/91961/Desktop/keerthana/Day_3/4.py =============
11
enter the n:
enter the n:
enter the n:
enter the n:
enter the n:
enter the n:
enter the n:
enter the n:
enter the n:
enter the n:
enter the n:
[1000]
>>> 
============= RESTART: C:/Users/91961/Desktop/keerthana/Day_3/4.py =============
[0, 1, 8, 27, 64, 125, 216, 343, 512, 729, 1000]
>>> 
============= RESTART: C:/Users/91961/Desktop/keerthana/Day_3/4.py =============
enter number:1223
Traceback (most recent call last):
  File "C:/Users/91961/Desktop/keerthana/Day_3/4.py", line 26, in <module>
    print(abs(odd=even))
TypeError: abs() takes no keyword arguments
>>> 
============= RESTART: C:/Users/91961/Desktop/keerthana/Day_3/4.py =============
enter number:1223
2
>>> 
============= RESTART: C:/Users/91961/Desktop/keerthana/Day_3/4.py =============
enter number:4567
2
>>> 
============= RESTART: C:/Users/91961/Desktop/keerthana/Day_3/4.py =============
enter number:123456789
5
>>> 
============= RESTART: C:/Users/91961/Desktop/keerthana/Day_3/4.py =============
[0, 1, 8, 27, 64, 125, 216, 343, 512, 729, 1000]
>>> 
============= RESTART: C:/Users/91961/Desktop/keerthana/Day_3/4.py =============
[0, 1, 8, 27, 64, 125, 216, 343, 512, 729, 1000]
>>> 
============= RESTART: C:/Users/91961/Desktop/keerthana/Day_3/4.py =============
enter number:16
Traceback (most recent call last):
  File "C:/Users/91961/Desktop/keerthana/Day_3/4.py", line 35, in <module>
    num=[int(d) for i in str(input("enter number:"))]
  File "C:/Users/91961/Desktop/keerthana/Day_3/4.py", line 35, in <listcomp>
    num=[int(d) for i in str(input("enter number:"))]
NameError: name 'd' is not defined
>>> 
============= RESTART: C:/Users/91961/Desktop/keerthana/Day_3/4.py =============
enter the term16
6560
>>> 
============= RESTART: C:/Users/91961/Desktop/keerthana/Day_3/4.py =============
enter the term13
64
>>> 
============= RESTART: C:/Users/91961/Desktop/keerthana/Day_3/4.py =============
enter the term16
2187
>>> 